assets/ros/core-monitor/style.css

```diff
@@ -20,11 +20,11 @@ table td, table th {
 }
 
 table tbody tr {
-  border-bottom: 1px solid #333;
+  border-bottom: 1px solid var(--vscode-sideBarSectionHeader-background);
 }
 
 table tbody tr:hover {
```

Comment:
it seems we might be removing the hover effect if we use the same color, but this is minor